bin/cli.js

```diff
@@ -70,6 +70,21 @@ For more information, see https://webpack.js.org/api/cli/.`);
 		try {
 			options = require("./utils/convert-argv")(argv);
 		} catch (err) {
+			if (err.code === "MODULE_NOT_FOUND") {
+				let errorMessage =
+					"\n\u001b[31mwebpack not installed, consider installing it using \n\u001b[32mnpm install --save-dev webpack\n";
+
+				if (process.env.npm_execpath !== undefined && process.env.npm_execpath.includes("yarn")) {
+					errorMessage =
+						"\n\u001b[31mwebpack not installed, consider installing it using \n\u001b[32myarn add webpack --dev\n";
+				}
+
+				console.error(errorMessage);
+				Error.stackTraceLimit = 1;
+				process.exitCode = 1;
+				return;
```

Comment:
I guess this is fine, as long as we're doing it in the scope of only `MODULE_NOT_FOUND`